_posts/2023-06-23-run-and-debug-rust-verification-harnesses-natively-on-vscode.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title:  "Run and debug Rust verification harnesses natively on VS Code"
+---
+
+Kani is a verification tool that can help you prove properties about your Rust code. To learn more about Kani, check out the [Kani tutorial](https://model-checking.github.io/kani/kani-tutorial.html) and our [previous blog posts](https://model-checking.github.io/kani-verifier-blog/).
```

Comment:
```suggestion
Kani is a verification tool that can help you systematically test properties about your Rust code. To learn more about Kani, check out the [Kani tutorial](https://model-checking.github.io/kani/kani-tutorial.html) and our [previous blog posts](https://model-checking.github.io/kani-verifier-blog/).
```

Reply:
We have used the word in previous blog posts including the introduction blog post, so I thought we can continue that for continuity.

Reply:
Resolved offline